performance-tests/bench_2/test_controller/unit_tests/unit_tests.mpc

```diff
@@ -0,0 +1,17 @@
+project (*ScenarioAllocationTest): dcps, dcpsexe, dcps_test, dcps_rapidjson, googletest {
```

Comment:
Something's still not quite right here, some of the VS2017 and VS2019 debug builds are failing. Take a look at the bench_builder_exe and bench_exe mpb files in performance-tests/bench_2 for hints on what might be missing?